automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioPage.scala

```diff
@@ -45,7 +45,7 @@ class RStudioPage(override val url: String)(implicit override val authToken: Aut
   def withRShinyExample[T](exampleName: String)(testCode: RShinyPage => T): T = {
     // Enter commands to launch the shiny app
     switchToNewTab {
-      val launchCommand = s"""shiny::runExample("$exampleName", launch.browser = T)"""
+      val launchCommand = "shiny::runExample(" + '"' + exampleName + '"' + ", launch.browser = T)"
```

Comment:
Is this actually any different? Seems this would result in the same string:
```
scala> val exampleName = "foo"
exampleName: String = foo

scala> val launchCommand1 = s"""shiny::runExample("$exampleName", launch.browser = T)"""
launchCommand1: String = shiny::runExample("foo", launch.browser = T)

scala> val launchCommand2 = "shiny::runExample(" + '"' + exampleName + '"' + ", launch.browser = T)"
launchCommand2: String = shiny::runExample("foo", launch.browser = T)

scala> launchCommand1 == launchCommand2
res0: Boolean = true
```

Reply:
Correct, I don't think the issue was with the string itself. I tried this change to see if maybe how the string was created would result in it being processed different. Looking that this has errored out on the most recent retest, it does seem like it does not avoid the issue.